apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerDataAdapter.cs

```diff
@@ -155,6 +161,11 @@ IDbCommand IDbDataAdapter.UpdateCommand
         }
 
         /// <summary>
+        /// The set of primary keys defined for <see cref="AutoGeneratedCommandTable"/>.
+        /// </summary>
+        public HashSet<string> AutoGeneratedCommandPrimaryKeys { get; } = new HashSet<string>();
```

Comment:
changed to an ISet -- this was part of last weeks changes actually.
It has to be public because the developer may use the default constructor and would then need a way to set the primary keys used for the autobuilt delete command.
Its far from ideal and I've asked spanner folk numerous times to give me a way to determine the primary key -- but all I have is the raw DDL create table statement.  ugh.